main.py

```diff
@@ -29,6 +29,7 @@
 listReg = f"{START_COMMENT}[\\s\\S]+{END_COMMENT}"
 
 waka_key = os.getenv('INPUT_WAKATIME_API_KEY')
+waka_url = os.getenv('INPUT_WAKATIME_URL', "https://wakatime.com") 

```

Comment:
As far as you can see, for now the environmental variables defaults are defined in `action.yml`. I understand your will to add defaults here (I hope to do it one day as well), but for now I guess it would be much cooler to put it into `action.yml`.